services/api/src/rsufwdsnmpset.py

```diff
@@ -331,5 +333,5 @@ def delete(request):
   if errors:
     return f"The provided args does not match required values: {str(errors)}", 400
   
-  response, code = config_del(request['rsu_ip'], request['manufacturer'], request['snmp_creds'], request['args']['msg_type'], request['args']['rsu_index'])
+  response, code = config_del(request['rsu_ip'], request['snmp_version'], request['snmp_creds'], request['args']['msg_type'], request['args']['rsu_index'])
```

Comment:
We could start using named arguments for these longer methods. Makes them more easily readable

Reply:
This has been addressed to the way you suggested.